app/shop.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+
+
+class Shop:
+    def __init__(self, name: str, location: list[int], products: dict) -> None:
+        self.name = name
+        self.location = location
+        self.products = products
+
+    def count_product(self, cust_cart: dict) -> list:
+        total = [val * self.products[key] for key, val in cust_cart.items()]
+        return total
+
+    def bill(self, customer_name: str, customer_prod: dict) -> None:
```

Comment:
customer_prod
what is this? I do not understand what this variable mean. Choose better name